modules/ROOT/partials/configuration/toolbar_mode.adoc

```diff
@@ -82,16 +87,17 @@ tinymce.init({
 ----
 
 [[scrolling]]
-==== `Scrolling`
+==== Scrolling
 
 The scrolling toolbar mode is intended for touch screen devices.
 
 If the `toolbar_mode` is configured to `scrolling`, the overflow toolbar buttons will remain on the toolbar, but will be hidden from view. To access the overflow toolbar buttons, scroll the toolbar. To scroll the toolbar on a touch screen, use a swiping action. To scroll on a desktop device, hold shift and scroll with a mouse.
 
 A scrollbar will be shown on the toolbar on desktop devices and hybrid devices in desktop mode.
 
-image::{baseurl}/images/scrolling_toolbar.png[Scrolling Toolbar]
+image:scrolling_toolbar.png[Scrolling Toolbar]
```

Comment:
This change is incorrect as it's meant to be a block image, not an inline image.
```suggestion
image::scrolling_toolbar.png[Scrolling Toolbar]
```